### PR TITLE
Finish models

### PR DESCRIPTION
**Pull request by jones-sam**
We have finished the models, created all the references, and created the seeds. We also ended up implementing the references without using any polymorphic associations.

### CODE REVIEW

**Comment by gabrielecanepa** [comment]
[Create and connect other models](https://trello.com/c/JsHNjPGm/7-create-and-connect-other-models)

**Comment by gabrielecanepa** [comment]
[Create and connect other models](https://trello.com/c/JsHNjPGm/7-create-and-connect-other-models)

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,63 @@
+# frozen_string_literal: true
+
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
 #
 # Examples:
 #
```
Comment (gabrielecanepa) [inline db/seeds.rb:7]: Also update the seeds every time you change something. It's also the quickest way to test if everything is fine! 🧪

File: app/models/task.rb [hunk 2]
```diff
@@ -1,2 +1,6 @@
+# frozen_string_literal: true
+
 class Task < ApplicationRecord
+  belongs_to :creator, class_name: 'User'
+  belongs_to :helper, class_name: 'User'
```
[the remark this answers is not in the document]
Comment (jones-sam) [reply app/models/task.rb:5]: added!

File: db/migrate/20200409160053_add_references_to_users.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddReferencesToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :users, :task_application, foreign_key: true
+    add_reference :users, :task
```
[the remark this answers is not in the document]
Comment (jones-sam) [reply db/migrate/20200409160053_add_references_to_users.rb:6]: removed this migration
(everyone remember to drop the db and migrate again)

File: app/models/user.rb [hunk 4]
```diff
@@ -1,6 +1,12 @@
...
+
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  has_many :task_applications
+  has_many :creators, class_name: 'Task', foreign_key: 'creator_id'
+  has_many :helpers, class_name: 'Task', foreign_key: 'helper_id'
```
[the remark this answers is not in the document]
Comment (jones-sam) [reply app/models/user.rb:11]: I tested the code without these lines, and it makes it unable to check what tasks the user has created or helped with from the user model, so these lines are required.

**Pull request merged**